src/ploneintranet/workspace/workspacecontainer.py

```diff
@@ -15,3 +15,12 @@ class WorkspaceContainer(Container):
     A folder to contain WorkspaceFolders
     """
     grok.implements(IWorkspaceContainer)
+
+try:
```

Comment:
That kind of functionality sounds like it should happen via a generic adapter in ploneintranet.attachments, rather than requiring the workspace to provide the interface manually.